lib/models/location.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'locations',
+  serialize: function () {
```

Comment:
Above this line is where you could add a one to many relation to `movies`. [Docs](http://bookshelfjs.org/#many-to-many)